### ISSUE TITLE
NVDA doesn't announce texts referenced by multiple aria-labelledby properly on Edge browser

### ISSUE THREAD

**Issue by swidjaja** [nvaccess/nvda#12001]
### Steps to reproduce:
1. Open the attached arialabelledby.html file on Edge browser
[arialabelledby.html.txt](https://github.com/nvaccess/nvda/files/5804699/arialabelledby.html.txt)

2. Navigate to the "Show more" button

### Actual behavior:
NVDA announces the button as  "button Good Stuffs"

### Expected behavior:
NVDA announces the button as  "button Show more Good Stuffs"

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA Version 2020.3

#### Windows version:
Windows 10 Enterprise

#### Name and version of other software in use when reproducing the issue:
Microsoft Edge 44.18362.449.0

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If addons are disabled, is your problem still occuring?
Unknown

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by feerrenrut** [comment]
Technical info:
I suspect this due to the `labelled by content` metadata that is attached when the label is a sub-node of the element with `labelled-by`. Labels aren't explicitly read when the label is part of the content, since the content will be read anyway. However multiple labels wasn't taken into account, thus this looks likely to be a bug in NVDA.

**Comment by Adriani90** [comment]
This issue is only occuring when using UIA for edge and Chromium browsers, and it occurs only ind MS Edge but not in Chrome.
UIA for Chromium browsers is by default set to "only when necessary", and with this default setting the issue is not occuring at all.
cc: @gerald-hartig for triaging.
